test/LibraryManager.Test/VersionedLibraryNamingSchemeTest.cs

```diff
@@ -11,7 +11,10 @@ public class VersionedLibraryNamingSchemeTest
     {
         [DataTestMethod]
         [DataRow("jquery@3.3.1", "jquery", "3.3.1")]
-        [DataRow("@angular/cli@1.0.0", "@angular/cli", "1.0.0")]
+        [DataRow("@angular/cli@1.0.0", "cli", "1.0.0")]
```

Comment:
These are wholly wrong.  The expected name is "@angular/cli".

Reply:
Resolving this in favor of other comment closer to the code change.  Better place to hash it out.